interface/patient_file/summary/demographics.php

```diff
@@ -899,7 +905,7 @@ function setMyPatient() {
     <title><?php echo xlt("Dashboard{{patient file}}"); ?></title>
 </head>
 
-<body class="mt-1 patient-demographic bg-light">
+<body id="DEM" class="mt-1 patient-demographic bg-light">
```

Comment:
Not really a good idea to add the id to body. My PR where I added styles to the tabs.scss is prob the better place to do styling in demographic view.

Reply:
@sjpadgett
This is a tricky issue.  

There is a decent amount of styling that already exists in the code base that refers to id of "DEM".  Should I move all of that styling to tabs.scss and not have it depend on the id "DEM".  This might be my preferred choice, but I don't know if it would mess up other areas.  It would be easiest for me to do this after your PR is merged.  How long will that be?

OR
I can create a div nested just inside body with both id="DEM" class="mt-1 patient-demographic bg-light".  I would think that this would be redundant since it could have just been done to body.

Personally, I think that DEM should have been a class and not an ID in the first place.  But to do that change would require changing even more code, which I want to avoid.

Also, if it is bad to give the tag body an ID, I would think that it would be just as bad to give it a class.  Is it?

I tried to search internet about using ID's and Classes for body tag.  I could not find anything that said that either was bad.

Reply:
I merged my PR and also cherry picked it back v7.0 release.
Ids are unique to their scope where a class may be used more than once.

It's not bad to give the body tab an id but it's the DEM id I don't trust.